<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>board</class>
 <widget class="QWidget" name="board">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>612</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>938</width>
    <height>612</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>938</width>
    <height>612</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Simon</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="upperLeft">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>469</width>
     <height>306</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lowerLeft">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>306</y>
     <width>469</width>
     <height>306</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="upperRight">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>0</y>
     <width>469</width>
     <height>306</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lowerRight">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>306</y>
     <width>469</width>
     <height>306</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="boardBckgrnd">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>612</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>boardBckgrnd</zorder>
  <zorder>upperRight</zorder>
  <zorder>upperLeft</zorder>
  <zorder>lowerLeft</zorder>
  <zorder>lowerRight</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
